common/src/main/java/com/wynntils/wc/custom/item/properties/SkillPointProperty.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.custom.item.properties;
+
+import com.wynntils.mc.render.FontRenderer;
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.utils.objects.CustomColor;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import com.wynntils.wc.custom.item.properties.type.TextOverlayProperty;
+import net.minecraft.ChatFormatting;
+
+public class SkillPointProperty extends ItemProperty implements TextOverlayProperty {
+    private static final CustomColor TEXT_COLOR = CustomColor.fromChatFormatting(ChatFormatting.WHITE);
+    private final TextOverlay textOverlay;
+
+    public SkillPointProperty(WynnItemStack item) {
+        super(item);
+        item.setCount(1);
```

Comment:
it might be better to define this sort of "override" as an attribute of `TextOverlayProperty` and do the actual handling in the feature code

Reply:
Do you mean like `isInventoryText`? Or should I make another property type like `ItemCountProperty`?

Reply:
similar to `isInventoryText`, yes

Reply:
I went a different route. As I only found a way to implement it by iterating over the entire inventory to set the item counts to 1 on specific items or do it ever render event. But I do lack some knowledge of Artemis' codebase so please let me know if there is another way.